tests.py

```diff
@@ -18,27 +17,7 @@
 from xml.etree import ElementTree as ET
 
 
-if sys.version_info >= (3, 8):
-    from unittest import mock
-else:
-    # unittest.mock in 3.7 is too old to support
-    # all the features used in the test suite
-    import mock
-
-from check_manifest import rmtree
-
-
-CAN_SKIP_TESTS = os.getenv('SKIP_NO_TESTS', '') == ''
-
-
-try:
-    codecs.lookup('oem')
-except LookupError:
-    HAS_OEM_CODEC = False
-else:
-    # Python >= 3.6 on Windows
-    HAS_OEM_CODEC = True
-
+from unittest import mock
```

Comment:
This import needs to be moved above the `from xml.etree` import or isort will complain.

Also, I need to review my isort command-line because a complaint of the sort

> ERROR: /home/runner/work/check-manifest/check-manifest/tests.py Imports are incorrectly sorted and/or formatted.

is useless when it won't tell me which imports where and how to sort them.  It could show a diff or something.